tests/parametric/test_otel_env_vars.py

```diff
@@ -0,0 +1,182 @@
+            }
+        ],
+    )
+    def test_otel_attribute_mapping(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+
+        assert resp["dd_service"] == "test2"
+        assert resp["dd_env"] == "test1"
+        assert resp["dd_version"] == "5"
+        tags = resp["dd_tags"]
+        assert "foo:bar1" in tags
+        assert "baz:qux1" in tags
+
+    @pytest.mark.parametrize(
+        "library_env", [{"OTEL_TRACES_SAMPLER": "always_on",}],
+    )
+    def test_otel_traces_always_on(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+        assert float(resp["dd_trace_sample_rate"]) == 1.0
+
+    @pytest.mark.parametrize(
+        "library_env", [{"OTEL_TRACES_SAMPLER": "always_off",}],
+    )
+    def test_otel_traces_always_off(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+        assert float(resp["dd_trace_sample_rate"]) == 0.0
+
+    @pytest.mark.parametrize(
+        "library_env", [{"OTEL_TRACES_SAMPLER": "traceidratio", "OTEL_TRACES_SAMPLER_ARG": "0.1"}],
+    )
+    def test_otel_traces_traceidratio(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+        assert float(resp["dd_trace_sample_rate"]) == 0.1
+
+    @pytest.mark.parametrize(
+        "library_env", [{"OTEL_TRACES_SAMPLER": "parentbased_always_on",}],
+    )
+    def test_otel_traces_parentbased_on(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+        assert float(resp["dd_trace_sample_rate"]) == 1.0
+
+    @pytest.mark.parametrize(
+        "library_env", [{"OTEL_TRACES_SAMPLER": "parentbased_always_off",}],
+    )
+    def test_otel_traces_parentbased_off(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+        assert float(resp["dd_trace_sample_rate"]) == 0.0
+
+    @pytest.mark.parametrize(
+        "library_env", [{"OTEL_TRACES_SAMPLER": "parentbased_traceidratio", "OTEL_TRACES_SAMPLER_ARG": "0.1"}],
+    )
+    def test_otel_traces_parentbased_ratio(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+        assert float(resp["dd_trace_sample_rate"]) == 0.1
+
+    @pytest.mark.parametrize(
+        "library_env", [{"OTEL_TRACES_EXPORTER": "none"}],
+    )
+    def test_otel_traces_exporter_none(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+        assert resp["dd_trace_enabled"] == "false"
+
+    @missing_feature(context.library == "nodejs", reason="Not implemented")
+    @pytest.mark.parametrize(
+        "library_env", [{"OTEL_LOG_LEVEL": "debug"}],
+    )
+    def test_otel_log_level_to_debug_mapping(self, test_agent, test_library):
+        resp = test_library.get_tracer_config()
+        assert resp["dd_log_level"] == "debug"
+        assert resp["dd_trace_debug"] == "true"
+
+    @missing_feature(context.library == "nodejs", reason="Not implemented")
```

Comment:
nit: can we give a more specific reason here. Why/what is not implemented in node